### PR TITLE
Scripts, Hello World )

### PR DESCRIPTION
**Pull request by nialone**
I added the script tab. Unfortunately, to see default scripts, you need to erase the program settings in the applications.
If the ACCA is installed for the first time, then there is no need.

Also, points in the constriction to disable the sending of commands that can be crashing through the ACC service. This is a temporary implementation. I plan to expand AccConfig for additional settings so that it does not send all the data.

Everything else is fixation for fusion. Thx.

### CODE REVIEW

**Comment by MatteCarra** [comment]
Thanks for the pull request.
Could you fix the conflicts? Thanks a lot!

**Comment by nialone** [comment]
Ready. Check plz )

**Comment by nialone** [comment]
Ready. I do not understand how to edit the files in the browser. It is almost blind. )))

File: app/src/main/java/mattecarra/accapp/activities/AccConfigEditorActivity.kt [hunk 1]
```diff
@@ -59,6 +59,7 @@ class AccConfigEditorActivity : ScopedAppActivity(), NumberPicker.OnValueChangeL
         setContentView(binding.root)
 
         content = binding.contentAccConfigEditor
+        content = binding.content
```
Comment (nialone) [inline app/src/main/java/mattecarra/accapp/activities/AccConfigEditorActivity.kt]: wrong add.. sorry. Need `content = binding.contentAccConfigEditor` For:
`<include layout="@layout/content_acc_config_editor"  android:id="@+id/content_acc_config_editor"/>`

**Comment by MatteCarra** [comment]
Check how to populate a table after a migration: https://github.com/MatteCarra/AccA/blob/baa39eb1426ac1c0d7fa16afb0598607e89a55ea/app/src/main/java/mattecarra/accapp/database/AccaRoomDatabase.kt#L82

**Comment by nialone** [comment]
Thanks for the hint. I will definitely try.
This will delete the code from the prepopulateDb.

**Comment by MatteCarra** [comment]
The code in prepopulateDb should stay for users that install the app for the first time.
The code in the migration pass is for users that update the app

**Comment by nialone** [comment]
I understood. I have no way to roll back the base on the emulator. )) [In accumulation](https://github.com/nialone/AccA/commit/eb93762ca95762622ff7e5af40d420fb324daf3d)

**Comment by VR-25** [comment]
Really liking where this is going.
Keep it up!

**Pull request merged**